### DISCUSSION TITLE
How do I install GNUSTEP Apps like Chess.app

### DISCUSSION THREAD

**Discussion by rocheston** [trunkmaster/nextspace#403] [Q&A]
Please guide us - how to install GNUSTEP Apps like the Chess.app

**Comment by armm77** [comment]
Hi, @rocheston 

1. git clone https://github.com/gnustep/gap
2. cd  gap
3. make

and as root.

4. make install
 
![Captura de Pantalla 2021-12-23 a la(s) 2 21 47 p m](https://user-images.githubusercontent.com/3104815/147283760-93036628-4fa7-492e-8e86-61da77738b56.png)

**Comment by rocheston** [comment]
Wow, the image looks lovely.  Managed to install the apps. Thank you so much. Nextspace platform is amazing.

**Comment by trunkmaster** [comment]
This is not an NEXTSPACE issue. Moving this to "Discussions".
